### ISSUE TITLE
Time options

### ISSUE THREAD

**Issue by jnst** [lucagrulla/cw#20]
I am satisfied with the features of `cw` already, but if you want to more better to improve, I propose the `Time options` feature implementation. 

I used [awslogs](https://github.com/jorgebastida/awslogs) until recently, but the problem with this tool is that it depends on python 2.x version. (In that respect, `cw` implemented with `golang` is very smart!)

`awslogs` has a feature called `Time options`, which is useful for viewing the most recent log.
* https://github.com/jorgebastida/awslogs#time-options
  * `--start=2m`
  * `--start=1h`

The `tail` command of `cw` can specify UTC `start time` as the third argument, but in my environment the timezone is `GMT+09:00`, which is a bit confusing.

```bash
$ cw -p my_other_profile tail -f -t /ecs/my-server ecs/my-server/f276cdff-375d-4c65-a4d6-29afd025f221 2018-10-01T10:13:00
2018-10-01T19:13:13 - [2018-10-01 10:13:13.488]    0 ms -- /health/check [:GET]   200 [] "-" "-" ::ffff:10.0.0.174
2018-10-01T19:13:17 - [2018-10-01 10:13:17.282]    0 ms -- /health/check [:GET]   200 [] "-" "-" ::ffff:10.0.1.137
2018-10-01T19:14:13 - [2018-10-01 10:14:13.518]    0 ms -- /health/check [:GET]   200 [] "-" "-" ::ffff:10.0.0.174
2018-10-01T19:14:17 - [2018-10-01 10:14:17.312]    0 ms -- /health/check [:GET]   200 [] "-" "-" ::ffff:10.0.1.137
```
This is because JST(GMT+09:00) time is output with `-t` flag although I have specified time in UTC.
Therefore it is convenient to specify like `Time option`.

```bash
$ cw tail --follow --timestamp my-log-group my-log-stream-prefix 2m
```
or
```bash
$ cw tail --follow --timestamp --start 2m my-log-group my-log-stream-prefix
```

**Comment by lucagrulla** [comment]
Thanks.

In my opinion UTC is the only real sane way to work on logs and not get confused (although I appreciate it's challenging when there's a gap between UTC and local time).

If you want to send a PR i'll consider it, but I'm not planning to do this any time soon.

**Comment by jnst** [comment]
OK, I got it 👍

**Comment by lucagrulla** [comment]
ok I started working on the option to pass delta of time ("2h30m") as start and end time. 
It will be done. 

Absolute dates and time will remain UTC.

**Comment by lucagrulla** [comment]
Available from v2.0.0.
